test/settings.e2e.js

```diff
@@ -0,0 +1,240 @@
+const { startApp, stopApp, screenshotApp } = require('./utils')
+import En from '../lang/en.json'
+import Es from '../lang/es.json'
+
+describe('Tests for Settings page', () => {
+  let app
+
+  beforeAll(async () => {
+    app = await startApp()
+  })
+
+  afterAll(async () => {
+    await stopApp(app)
+  })
+
+  test('Clicking on "Settings" tab loads it up correctly', async () => {
+    await app.client
+      .$(`div=${En['Settings.Title']}`)
+      .click()
+      .pause(1000)
+
+    const labelTestOptionsVisible = await app.client.isVisible(
+      `h4=${En['Settings.TestOptions.Label']}`
+    )
+    const labelAutoTestingVisible = await app.client.isVisible(
+      `h4=${En['Settings.AutomatedTesting.Label']}`
+    )
+    const labelPrivacyVisible = await app.client.isVisible(
+      `h4=${En['Settings.Privacy.Label']}`
+    )
+
+    expect(labelTestOptionsVisible).toBe(true)
+    expect(labelAutoTestingVisible).toBe(true)
+    expect(labelPrivacyVisible).toBe(true)
+  })
+
+  test('Language changes to Spanish', async () => {
+    const languageLabelEn = await app.client.isVisible('label=Language')
+    expect(languageLabelEn).toBe(true)
+
+    const languageSelectValue = await app.client
+      .$('select[data-testid=select-language]')
+      .getValue()
+    expect(languageSelectValue).toBe('en')
+
+    // Clicking and selecting value instead of using using setValue() because
+    // webdriver thinks the <select> option is in a wrong state (uneditable)
+    await app.client.$('select[data-testid=select-language]').click()
+    await app.client.$('option=Spanish').click()
+
+    const languageLabelEs = await app.client.isVisible('label=Idioma')
```

Comment:
This can be `isESLabelVisible` or something. Then reading the assertion in the next line makes sense.

Reply:
Or use a different assertion than `toBe(true)`.

Reply:
Changed to `isEsLabelVisible`